stl/inc/algorithm

```diff
@@ -5223,6 +5350,26 @@ namespace ranges {
                 return {_STD move(_First), _STD move(_Output)};
             }
 
+#if _USE_STD_VECTOR_ALGORITHMS
+            if constexpr (is_same_v<_Pj, identity> && sized_sentinel_for<_Se, _It>
+                          && _Vector_alg_in_unique_is_safe<_It, _Out, _Pr> && _Iterator_is_contiguous<_Out>

```

Comment:
```
"C:\stlBuild\out\inc\algorithm", line 5355: error: too many arguments for
          variable template "std::_Vector_alg_in_unique_is_safe"
                            && _Vector_alg_in_unique_is_safe<_It, _Out, _Pr> && _Iterator_is_contiguous<_Out>
                                                                        ^
```

The error is correct:

https://github.com/microsoft/STL/blob/f2a2933dd65d9e8d3fa698a97b6074f7ef00e1fd/stl/inc/algorithm#L244-L245